tests/Datagrid/ListMapperTest.php

```diff
@@ -168,7 +168,7 @@ public function testAddOptionIdentifierWithWrongValue(bool $expected, $value): v
         $this->assertFalse($this->listMapper->has('fooName'));
 
         $this->expectException(\InvalidArgumentException::class);
-        $this->expectExceptionMessageMatches('/^Value for "identifier" option must be boolean, .+ given.$/');
+        $this->expectExceptionMessageRegExp('/^Value for "identifier" option must be boolean, .+ given.$/');
```

Comment:
@greg0ire changed this in https://github.com/sonata-project/SonataAdminBundle/pull/5899 because [`expectExceptionMessageRegExp` has been removed in PHPUnit 9](https://github.com/sebastianbergmann/phpunit-documentation-english/pull/182), `expectExceptionMessageMatches` was added in 8.4 or 8.5 I think.

Reply:
@franmomu @greg0ire But the phpunit-bridge still use the 8.3
https://github.com/symfony/phpunit-bridge/blob/master/bin/simple-phpunit.php#L98

Reply:
You can set the phpunit version with `SYMFONY_PHPUNIT_VERSION` env variable. This should be done in dev-kit, @phansys [already started it](https://github.com/sonata-project/dev-kit/pull/503).